app/controllers/spree/api/images_controller_decorator.rb

```diff
@@ -0,0 +1,15 @@
+Spree::Api::ImagesController.class_eval do
```

Comment:
do we really need to override the spree's controller here? Do we need any `before_filter` or any other dependency from that `Spree::Api::ImagesController`? I'd be great if we could have a simple single-action controller of our own so that we don't depend on Spree for this simple feature. What do you think?